### PR TITLE
Resolved buttons placing in desktop and mobile view Issue#2739

### LINKED ISSUE #2739: Buttons are fixed & Mobile view has styling issues on Consultation Page
https://care.coronasafe.in/facility/551f9e09-04a3-4f52-ade2-1db45588741d/patient/0eacd048-afcc-4a5e-a8b9-14ee7c581f25/consultation/9d1b307b-2069-4feb-aa7d-662984cb7a23

- [x] The 4 buttons are fixed
https://www.loom.com/share/76f0762156184060a3d73f35a6cdbf2d
- [x] The buttons are misplaced in the **mobile** view
Screenshot (390)

### PR DESCRIPTION
**Pull request by Pranshu1902**
Fixes #2739 

1. Resolved fixed buttons issue
https://www.loom.com/share/8e4e32c7774c4c8194c72f30a0adb45e

2. Resolved styling between buttons in mobile view
Screenshot (391)

### CODE REVIEW

**Review by developedBySJ — CHANGES_REQUESTED** [review]
@Pranshu1902 Can you make full-width buttons and add spacing above buttons?

**Comment by Pranshu1902** [comment]
Updated mobile view:
Screenshot (392)

File: src/Components/Facility/ConsultationDetails.tsx [hunk 1]
```diff
@@ -420,7 +419,7 @@ export const ConsultationDetails = (props: any) => {
             className="sm:m-0 sm:p-0"
             breadcrumbs={true}
           />
-          <div className="lg:absolute right-0 top-0 flex sm:flex-row sm:items-center flex-col space-y-1 sm:space-y-0 sm:divide-x-2">
+          <div className="right-0 top-0 flex sm:flex-row sm:items-center flex-col space-y-1 sm:space-y-0 sm:divide-x-2">
```
Comment (developedBySJ) [inline src/Components/Facility/ConsultationDetails.tsx]: You can use [`gap`](https://tailwindcss.com/docs/gap) classes for spacing

**Review by developedBySJ — APPROVED** [review]
LGTM

**Review by cp-Coder — APPROVED** [review]

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Comment by khavinshankar** [comment]
@Pranshu1902 please do work on the merge conflicts

**Comment by Pranshu1902** [comment]
@khavinshankar done!

**Review by cp-Coder — APPROVED** [review]

**Comment by nihal467** [comment]
@Pranshu1902 



the buttons are conjuested

**Comment by Pranshu1902** [comment]
@nihal467 bug resolved!
Desktop view:
image

Mobile View:
image

**Comment by nihal467** [comment]
@Pranshu1902 test failed

- [ ] the button is not in uniform size compared to others
- [ ] verified by is misaligned, it should be coming on the left side

**Comment by Pranshu1902** [comment]
@nihal467 
Updated view:
image

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**